<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMap</class>
 <widget class="QDialog" name="NewMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>54</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>53</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="blend_image_width">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="blend_image_height">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>32</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>512</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox_19">
     <property name="title">
      <string>Terrain</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QRadioButton" name="map_size">
           <property name="text">
            <string>Map size</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="map_width">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>128</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>256</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>512</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1024</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2048</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4096</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8192</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="map_height">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>128</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>256</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>512</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1024</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2048</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4096</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8192</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="image">
           <property name="text">
            <string>Image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="image_name">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>6</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="image_load">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>load</string>
             </property>
             <property name="icon">
              <iconset resource="icons/icons.qrc">
               <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="diffuse_textures">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Diffuse textures</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QComboBox" name="terrain_diffuse_texture_0"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_diffuse_texture_1"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_diffuse_texture_2"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_diffuse_texture_3"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_diffuse_texture_4"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="normal_textures">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Normal textures</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="terrain_normal_texture_0"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_normal_texture_1"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_normal_texture_2"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_normal_texture_3"/>
         </item>
         <item>
          <widget class="QComboBox" name="terrain_normal_texture_4"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/gtk-apply.png</normaloff>:/icons/gtk-apply.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons/gtk-cancel.png</normaloff>:/icons/gtk-cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Blend image size</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>clicked()</signal>
   <receiver>NewMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>NewMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>map_size</sender>
   <signal>toggled(bool)</signal>
   <receiver>image_load</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>map_size</sender>
   <signal>toggled(bool)</signal>
   <receiver>image_name</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>image</sender>
   <signal>toggled(bool)</signal>
   <receiver>map_height</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>image</sender>
   <signal>toggled(bool)</signal>
   <receiver>map_width</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>map_size</sender>
   <signal>clicked(bool)</signal>
   <receiver>ok</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
